config.json

```diff
@@ -646,12 +646,12 @@
     "msbuild": {
       "osSpecific": {
         "windows": {
-          "defaultParameters": "/nologo /verbosity:minimal /clp:Summary /maxcpucount /nodeReuse:false /l:BinClashLogger,Tools\\net46\\Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log",
-          "path": "Tools/msbuild.cmd"
+          "defaultParameters": "msbuild /nologo /verbosity:minimal /clp:Summary /maxcpucount /nodeReuse:false /l:BinClashLogger,Tools\\Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log",
+          "path": "Tools/dotnetcli/dotnet"
```

Comment:
Shouldn't we still be calling msbuild.cmd and simply changing that to instead use dotnet msbuild? I get that would be a buildtools change but I think that this is what we should do in both cases (Unix and Windows) as opposed to having this change here. Specially because that way you can always just call msbuild.cmd/sh and that would resolve the right host plus you could add extra variables there like disabling the firstTimeExperience and disable the multilevel lookup.

Reply:
I had a similar reaction but in the end it doesn't matter we will soon be doing away with this entire config.json file and run.exe.

Reply:
Also - changing this in buildtools would affect coreclr, wcf and core-setup repos. I'm not equipped to try to change those repos at this time.